packages/web-components/src/counter-badge/counter-badge.stories.ts

```diff
@@ -39,84 +38,164 @@ export default {
   },
   argTypes: {
     appearance: {
+      description: 'Sets the appearance of the badge to one of the predefined styles',
       options: Object.values(CounterBadgeAppearance),
-      control: {
-        type: 'select',
+      mapping: { '': null, ...CounterBadgeAppearance },
```

Comment:
For this and probably all `mapping` in CounterBadge, we probably need a corresponding empty `''` value in `options`?